### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by kingflurkel** [nicoSWD/put.io-adder#19]
Just wanted to say thanks, I've been using this little app forever.

**Comment by joshkerr** [comment]
Would like to echo this person's appreciation for the app. Thank you. Its been working beautifully since version 3.0. I can't really think of much to improve. Maybe make the app live in the menu bar instead of the dock?

**Comment by nicoSWD** [comment]
Thanks guys! :) @kingflurkel @joshkerr 

It really means a lot! I have some ideas for improvement, but unfortunately not a lot of time at the moment. I'd like to add an option to download files from put.io, which shouldn't be a too hard to do. And something that allows you to search for downloaded files. This is actually half implemented, but hidden for now as it's not working very smoothly.

I'm always open to pull requests, if someone with more time on their hands wants to help.

Thanks again!

**Comment by matt-sawyers** [comment]
Agreed. I really like the simplicity of the application @nicoSWD Thank you so much for working on it. I know it's not an easy task to make time to work on these types of projects, but I appreciate it :)

**Comment by nicoSWD** [comment]
Thanks again! :) It really means a lot. @matt-sawyers

**Comment by meat-ball** [comment]
app is awesome!

**Comment by azinman** [comment]
+1

**Comment by jorgenbs** [comment]
+1 love it

**Comment by nicoSWD** [comment]
❤️ 

@meat-ball  @azinman @jorgenbs
